kernal/drivers/x16/memory.s

```diff
@@ -271,6 +277,35 @@ jsrfar:
 __jmpfr:
 	jmp $ffff
 
+.pushcpu
+.setcpu "65816"
+
+.segment "KJFAR816"
+.assert * = jsrfar3n, error, "jsrfar3n must be at specific address"
```

Comment:
Does it? Why can't this be exported?

Reply:
In this case was matching the behavior of jsrfar3.  Not sure why it was like that before

Reply:
Probably because `banks.inc` is hardcoding the identifiers (I guess bank compilation shenanigans need it? :/)

Reply:
Ah yes, this is exactly why.  Other banks include `jsrfar.inc` and need those symbols.  Seems a bit easier to keep using this logic.